components/DataTable.js

```diff
@@ -67,47 +67,47 @@ const Container = styled.div`
       border-top: 1px solid ${props => props.theme.colors.gray};
       padding: 3rem 0;
 
-      @media (min-width: ${props => props.theme.medium}) {
+      @media (min-width: ${props => props.theme.breakpoints.medium}) {
         display: flex;
       }
 
       .dataset-title {
         color: ${props => props.theme.colors.black};
-        font-weight: ${props => props.theme.fontWeights.bold};
-        letter-spacing: ${props => props.theme.letterSpacings.medium};
+        font-weight: ${props => props.theme.typography.weights.bold};
+        letter-spacing: ${props => props.theme.typography.letter_spacings.headings.medium};
       }
 
       .dataset-date {
-        font-size: ${props => props.theme.fontSizes.sm};
-        line-height: ${props => props.theme.lineHeights.sm};
+        font-size: ${props => props.theme.typography.sizes.body.small};
+        line-height: ${props => props.theme.typography.line_heights.body.small};
```

Comment:
I'm more used to seeing camel case in JavaScript that snake case. Any reason this should be `line_heights` instead of `lineHeights`? (same q for `letter_spacings`)

Reply:
I was back and forth on this and I'm of two minds here.

Personally, I find camel case harder to read and I could see a case for making JavaScript variables related to CSS properties underscored to distinguish them from other variables. 

On the other hand, camel case is more standard as you said, we are already using it throughout and it probably feels a bit better when using dot notation as well.

I'm open to either way, it won't be a big refactor for me to convert all of these to camel case. Let me know what you think.

Reply:
Agree with all of that! I'm cool with either as long as we're consistent. 👍